### PR TITLE
Fix typesetting issues in documentation "Introduction to Splines"

### PR DESCRIPTION
**Pull request by markriegler**
In the documentation "Introduction to Splines", it fixes issues with subscripts and brackets. Additionally, some minor changes to the text like a missing hyphen are made.

### CODE REVIEW

File: docs/markdown/spline_intro.md [hunk 1]
```diff
@@ -81,7 +85,7 @@ The resulting spline curve can then be computed as follows:
 
 .. math::
 
-   \mathbf{S}(u)=(1-u)^{2}\mathbf{P}\_{1}+2u (1-u)\mathbf{P}\_{2}+u^{2}\mathbf{P}\_{3}
+   \mathbf{S}(u)=(1-u)^{2}\mathbf{P}_{1}+2u (1-u)\mathbf{P}_{2}+u^{2}\mathbf{P}_{3}
```
Comment (j042) [inline docs/markdown/spline_intro.md:88]: is it just me or did you get this too? maybe a space after {1}?
Screenshot 2024-05-17 at 15 06 58

File: docs/markdown/spline_intro.md [hunk 2]
```diff
@@ -93,7 +97,7 @@ Bézier curves have the following properties:
 
 .. math::
 
-  N_{1,p}(0)&=\underbrace{\frac{p!}{0!p!}}\_{=1}\underbrace{0^{0}}\_{=1}\underbrace{(1-0)^{p}}\_{=1}=1
+  N_{1,p}(0)&=\underbrace{\frac{p!}{0!p!}}_{=1}\underbrace{0^{0}}_{=1}\underbrace{(1-0)^{p}}_{=1}=1
```
Comment (j042) [inline docs/markdown/spline_intro.md:100]: Screenshot 2024-05-17 at 15 07 38
this one too

File: docs/markdown/spline_intro.md (hunk 1 above)
Comment (markriegler) [reply docs/markdown/spline_intro.md:88]: For me, I don't get these issues. For me it displays this:
Screenshot 2024-05-17 at 16 13 29

File: docs/markdown/spline_intro.md (hunk 2 above)
Comment (markriegler) [reply docs/markdown/spline_intro.md:100]: And this:
Screenshot 2024-05-17 at 16 16 11

File: docs/markdown/spline_intro.md [hunk 3]
```diff
@@ -183,10 +187,10 @@ Subsequently, the degree of the basis function can be raised bit by bit using
 until the desired degree has been reached.
 
 
-####    Properties of B-spline basis functions:
+#### Properties of B-spline basis functions:
 -  constitute a partition of unity :math:`\sum_{i=1}^{n}N_{i,p}(u)=1`
 -  non-negative
--  continuously differentiable within knot spans, :math:`p\cdots` times differentiable across knots (multiplicity of knot)
+-  continuously differentiable within knot spans, :math:`p` times differentiable across knots (multiplicity of knot)
```
Comment (jzwar) [inline docs/markdown/spline_intro.md:193]: Not sure why this is missing here, but could you add that the basis functions attain exactly one maximum on the interval [t_p+1, t_n+1] and that the same properties also hold for their rational counterparts.

**Review by jzwar — APPROVED** [review]
LGTM, thanks for fixing the typos

File: docs/markdown/spline_intro.md (hunk 2 above)
Comment (jzwar) [reply docs/markdown/spline_intro.md:100]: cannot reproduce error in local build @j042 on MacBook

Comment (j042) [reply docs/markdown/spline_intro.md:100]: thanks for checking. Works fine after updating the dependencies

File: docs/markdown/spline_intro.md (hunk 3 above)
Comment (j042) [reply docs/markdown/spline_intro.md:193]: (except zero-degree)

File: docs/markdown/spline_intro.md [hunk 4]
```diff
@@ -208,6 +218,15 @@ The NURBS basis functions :math:`R_{i,p}` are computed using the B-spline basis
 
 Again, :math:`\omega_i` is a weight.
 
+
+#### Properties of NURBS basis functions:
+
+The same properties of B-spline basis functions hold with the following additions:
+
+- computation of a set of basis functions additionally requires the weights :math:`\{\omega_i\}`
+- each basis function attains exactly one maximum in the interval :math:`[0,1]`, except for the case :math:`p=0`
```
Comment (j042) [inline docs/markdown/spline_intro.md:227]: shouldn't `[0, 1]` be the same as above?

Comment (markriegler) [reply docs/markdown/spline_intro.md:227]: I think so. I'll correct it

**Review by j042 — APPROVED** [review]
Awesome, thank you!

**Pull request merged**